lib/IDE/Refactoring.cpp

```diff
@@ -2602,6 +2602,106 @@ bool RefactoringActionLocalizeString::performChange() {
   return false;
 }
 
+static void generateMemberwiseInit(SourceEditConsumer &EditConsumer,
+                            SourceManager &SM,
+                            SmallVectorImpl<std::string>& memberNameVector,
+                            SmallVectorImpl<std::string>& memberTypeVector,
+                            SourceLoc targetLocation) {
+  
+  if (memberTypeVector.size() > 0) {
```

Comment:
Address the previous comment and we don't need to check `memberTypeVector` isn't empty here. Instead, we can add:
```
assert(!memberTypeVector.empty());
assert(memberTypeVector.size() == memberNameVector.size());
```